packages/block-library/package.json

```diff
@@ -59,6 +59,7 @@
 		"memize": "^1.1.0",
 		"moment": "^2.22.1",
 		"react-easy-crop": "^3.0.0",
+		"reakit": "1.1.0",
```

Comment:
**Non-blocking:**

Do you happen to know if we have a convention about the caret operator for npm semantic versioning?

Reply:
Im not sure about any specific convention, but I noticed this package is also missing the caret in the `block-editor`'s package.json so it seems ok to go with the exact versioning for it.

Reply:
Gotcha. Resolving this conversation because formalizing a convention will be outside the scope of this PR.